lib/web_ui/lib/src/engine/text_editing/composition_aware_mixin.dart

```diff
@@ -0,0 +1,93 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:html' as html;
+
+import 'text_editing.dart';
+
+/// Provides default functionality for listening to HTML composition events.
```

Comment:
Nit: Should we say "browser" composition events instead of HTML?  I don't know if the events are related to HTML exactly, or what spec they come from.

Reply:
Same for below where "HTML" is repeated.

Reply:
I see it's called `html.EventListener` in the code, maybe it's fine as-is then.